state/postgres_statement.go

```diff
@@ -20,34 +20,32 @@ type PostgresStatement struct {
-	LocalBlksHit      int64   // Total number of local block cache hits by the statement
-	LocalBlksRead     int64   // Total number of local blocks read by the statement
-	LocalBlksDirtied  int64   // Total number of local blocks dirtied by the statement
-	LocalBlksWritten  int64   // Total number of local blocks written by the statement
-	TempBlksRead      int64   // Total number of temp blocks read by the statement
-	TempBlksWritten   int64   // Total number of temp blocks written by the statement
-	BlkReadTime       float64 // Total time the statement spent reading blocks, in milliseconds (if track_io_timing is enabled, otherwise zero)
-	BlkWriteTime      float64 // Total time the statement spent writing blocks, in milliseconds (if track_io_timing is enabled, otherwise zero)
-
-	// Postgres 9.5+
-	MinTime    null.Float // Minimum time spent in the statement, in milliseconds
-	MaxTime    null.Float // Maximum time spent in the statement, in milliseconds
-	MeanTime   null.Float // Mean time spent in the statement, in milliseconds
-	StddevTime null.Float // Population standard deviation of time spent in the statement, in milliseconds
+	Calls             int64      // Number of times executed
+	TotalTime         float64    // Total time spent in the statement, in milliseconds
+	Rows              int64      // Total number of rows retrieved or affected by the statement
+	SharedBlksHit     int64      // Total number of shared block cache hits by the statement
+	SharedBlksRead    int64      // Total number of shared blocks read by the statement
+	SharedBlksDirtied int64      // Total number of shared blocks dirtied by the statement
+	SharedBlksWritten int64      // Total number of shared blocks written by the statement
+	LocalBlksHit      int64      // Total number of local block cache hits by the statement
+	LocalBlksRead     int64      // Total number of local blocks read by the statement
+	LocalBlksDirtied  int64      // Total number of local blocks dirtied by the statement
+	LocalBlksWritten  int64      // Total number of local blocks written by the statement
+	TempBlksRead      int64      // Total number of temp blocks read by the statement
+	TempBlksWritten   int64      // Total number of temp blocks written by the statement
+	BlkReadTime       float64    // Total time the statement spent reading blocks, in milliseconds (if track_io_timing is enabled, otherwise zero)
+	BlkWriteTime      float64    // Total time the statement spent writing blocks, in milliseconds (if track_io_timing is enabled, otherwise zero)
+	MinTime           null.Float // Minimum time spent in the statement, in milliseconds
+	MaxTime           null.Float // Maximum time spent in the statement, in milliseconds
+	MeanTime          null.Float // Mean time spent in the statement, in milliseconds
+	StddevTime        null.Float // Population standard deviation of time spent in the statement, in milliseconds
```

Comment:
Should we turn these into regular `float64` values now? (or is there ever a case where they are NULL on 9.5+?)

Reply:
9.5+ should work. If someone is using a super-old un-upgraded 1.2 from Postgres 9.4, this would break. It's probably okay to drop support for that, too.

Reply:
I changed these to raw floats, and to my surprise, nothing broke. Do we actually [do anything with these](https://github.com/pganalyze/collector/blob/main/output/transform/postgres_statements.go#L46-L66) right now?

Reply:
... doesn't look like it's used at all? 🤔

Reply:
Got it. So we could do something like plot the latest average, but that's probably not very useful because right now it becomes more and more weighted toward the past: the number grows in inertia and will stop responding meaningfully to even dramatic changes, right?